packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -1007,20 +665,22 @@ contract('FederatedAttestations', (accounts: string[]) => {
         identifier1,
         issuer1,
         account1,
-        nowUnixTime,
         signer1,
+        nowUnixTime,
         sig.v,
         sig.r,
         sig.s
       )
+      const publishedOn = Math.floor(Date.now() / 1000)
```

Comment:
I think this check may end up being flakey depending on when the devchain publishes this (I had this fail once locally but passes most of the time). Can we check that `publishedOn` falls within a reasonable window instead?

Reply:
got around this by fetching the onchain `publishedOn` value to check against - basically not testing for anything

Reply:
nice! can you add a comment mentioning why? (so someone in the future doesn't get super confused about this) you can also do a separate check after the log check, checking that publishedOn >= `issuedOn` in this test case to provide a semi-reasonable bound/sanity check!